api-tests/features/p2-f2f-journey.feature

```diff
@@ -22,6 +22,65 @@ Feature: P2 F2F journey
     When I start a new 'medium-confidence' journey
     Then I get a 'page-ipv-pending' page response
 
+  Scenario: Pending F2F request delete identity
+    # Initial journey
+    Given I start a new 'medium-confidence' journey
```

Comment:
Is there a way of summing this up into one step? Similar to `the subject already has the following credentials` but for an entire identity. (It could be an extension of the same method)

Reply:
This would be a bit trickier, as it would need to create the pending F2F record which is private to IPV Core (unlike EVCS). TBH given the hairiness around F2F it's probably not a bad thing to run through them properly, within reason.

What could be useful is putting this in a `Background` section for the tests that all require a pending F2F identity. (You can use a `Rule` if you need to group some of the scenarios to do this)

Reply:
Thanks, updated it. Moved common steps in background and rule :)